ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -0,0 +1,273 @@
+            NewFeatures(k,37)=mean(abs(CIFTI.cdata(CORTEX_start_idx:CORTEX_LEFT_end_idx,k)))./mean(abs(CIFTI.cdata(CORTEX_RIGHT_end_idx:CEREBELLUM_end_idx,k))); %Left Cerebral Cortex/Right Cerebral Cortex
+            NewFeatures(k,38)=mean(abs(Volume_data(wm_data==1,k)))./mean(abs(Volume_data(:,k))); %2 Voxel Eroded WM/Volume
+            NewFeatures(k,39)=mean(abs(Volume_data(csf_data==1,k)))./mean(abs(Volume_data(:,k))); %2 Voxel Eroded CSF/Volume
+            NewFeatures(k,40)=mean(abs(Volume_data(edge_data==1,k)))./mean(abs(Volume_data(:,k))); %Edge/Volume (CSF outside brain)
+            NewFeatures(k,41)=mean(abs(Volume_data(dropout_data>0.5,k)))./mean(abs(Volume_data(:,k))); %Dropout/Volume (computed using HCP Pipelines from SE and GRE Images)
+            NewFeatures(k,42)=mean(abs(CIFTI.cdata(CIFTIDropouts.cdata>0.25,k)))./mean(abs(CIFTI.cdata(:,k))); %CIFTIDropout/CIFTI (computed using HCP Pipelines from SE and GRE Images)
+            NewFeatures(k,43)=mean(abs(Volume_data(noisy_data==1,k)))./mean(abs(CIFTI.cdata(:,k))); %NoisyVoxels/CIFTI (top 12.5% variance)
+            NewFeatures(k,44)=mean(abs(CIFTIMSMAll.cdata(VisualROI==1,k)))./mean(abs(CIFTIMSMAll.cdata(NonVisualROI==1,k))); %VisualCortexCIFTI/NonVisualCortexCIFTI
+            NewFeatures(k,45)=mean(abs(CIFTIMSMAll.cdata(VisualROI==1,k)))./mean(abs(CIFTIMSMAll.cdata(CEREBELLUM_start_idx:CEREBELLUM_end_idx,k))); %VisualCortexCIFTI/CerebellumCIFTI              
+            NewFeatures(k,46)=mean(abs(CIFTIMSMAll.cdata(LanguageROI==1,k)))./mean(abs(CIFTIMSMAll.cdata(NonLanguageROI==1,k))); %LanguageCortexCIFTI/NonLanguageCortexCIFTI
+            NewFeatures(k,47)=mean(abs(CIFTI.cdata(:,k)))./mean(abs(Volume_data(NonGrey==1,k))); %CIFTI/NonGrey
+            NewFeatures(k,48)=mean(abs(CIFTI.cdata(1:CEREBELLUM_end_idx,k)))./mean(abs(Volume_data(NonGrey==1,k))); %Cerebral Cortex/NonGrey
+            NewFeatures(k,49)=mean(abs(CIFTI.cdata(:,k)))./mean(abs(Volume_data(NonGreyPlusEdge==1,k))); %CIFTI/NonGreyPlusEdge
+            NewFeatures(k,50)=mean(abs(CIFTI.cdata(1:CEREBELLUM_end_idx,k)))./mean(abs(Volume_data(NonGreyPlusEdge==1,k))); %Cerebral Cortex/NonGreyPlusEdge
+            NewFeatures(k,51)=mean(abs(CIFTIMSMAllSmooth.cdata(:,k)))./mean(abs(VolumeSmooth_data(NonGrey==1,k))); %CIFTI/NonGrey
+            NewFeatures(k,52)=mean(abs(CIFTIMSMAllSmooth.cdata(1:CEREBELLUM_end_idx,k)))./mean(abs(VolumeSmooth_data(NonGrey==1,k))); %Cerebral Cortex/NonGrey
+            NewFeatures(k,53)=mean(abs(CIFTIMSMAllSmooth.cdata(:,k)))./mean(abs(VolumeSmooth_data(NonGreyPlusEdge==1,k))); %CIFTI/NonGreyPlusEdge
+            NewFeatures(k,54)=mean(abs(CIFTIMSMAllSmooth.cdata(1:CEREBELLUM_end_idx,k)))./mean(abs(VolumeSmooth_data(NonGreyPlusEdge==1,k))); %Cerebral Cortex/NonGreyPlusEdge
+
+            for l=1:max(CorticalParcellation.cdata)
```

Comment:
Please don't use `l` as a variable, particularly if it is numeric, it looks too much like a `1`.